### ISSUE TITLE
Toast's title is not alignment center

### ISSUE THREAD

**Issue by tuanphamanh91** [scalessec/Toast#53]
Hello, i work on iOS 8.1 and test on device. 
Toast's title is not alignment center, please help me. Thanks advance!
Here is my code:
```
            [self.view makeToast:@"You've unlocked from level 16 to level 30!"
                        duration:4.0
                        position:CSToastPositionTop
                           title:@"Congratulations"];
```
Here is image:
![Image of Yaktocat](http://upanhtocdo.com/image.php?di=I2V5)

**Comment by hidefguy** [comment]
Ditto - there should be a way to center-align the title; on short messages it's probably fine as is, on longer messages it looks ugly.

**Comment by adams0917** [comment]
Please help me how i can add text center alignment. This is urgent issue for me.

**Comment by scalessec** [comment]
@adams0917 I'm finishing up a new version of Toast (3.0) that supports custom styling like this. If you want to try it now, here's the branch: https://github.com/scalessec/Toast/tree/experimental/3.0

Here's how you do it with Toast 3.0:
```objc
CSToastStyle *style = [[CSToastStyle alloc] initWithDefaultStyle];
style.titleAlignment = NSTextAlignmentCenter;
[self.view makeToast:@"This is a piece of toast."
            duration:3.0
            position:CSToastPositionBottom
               style:style];
```

If you want the title always be centered for all toast views you can just do this:
```objc
CSToastStyle *style = [[CSToastStyle alloc] initWithDefaultStyle];
style.titleAlignment = NSTextAlignmentCenter;
[CSToastManager setSharedStyle:style];
```

Then you could just pass `nil` for the style parameter going forward and the title would always appear centered.

Please keep in mind though that the experimental/3.0 branch is still technically in development. I plan to have the official release out and available on CocoaPods by early next week. I'll close this issue once the release is officially out.

**Comment by adams0917** [comment]
Thanks for your reply. But there is only 2.4 version i cocoapods. Is it support Center alignment (2.4 version)?

**Comment by scalessec** [comment]
No, you'll have to wait until next week if you want to use CocoaPods. You could hack 2.4 to support center alignment by changing line 303 in `UIView+Toast.m` to use `NSTextAlignmentCenter`... But that's probably not the best idea unless you create your own fork.

**Comment by scalessec** [comment]
3.0 is now available in CocoaPods. I have a few more updates that I'm going to release in 3.1 later this week.

**Comment by jhw8610** [comment]
Hey Charles, just a heads up, setting `textAlignment` to `NSTextAlignmentCenter` only sets the text alignment within the label frame. This means that if the title or message is shorter than the width of the wrapper view, it does not appear to be centered.

One way to fix this is to set `'UILabel'.center.x` to `wrapperView.center.x`.

**Comment by scalessec** [comment]
@jhw8610, good catch. Sorry about that. I created a new issue (#63), and I'll include this fix in the next release (3.1) that I'm planning in the next few days. Let me know if you come up with any other issues related to style.
